### ISSUE TITLE
README.md test app credential links to "Forbidden" URL

### ISSUE THREAD

**Issue by mr-c** [CenterForOpenScience/osf.io#4433]
"For local development, the COS provides [test app credentials](https://osf.io/m2hig/wiki/home/) for a number of services."

https://osf.io/m2hig/wiki/home/:

```
Forbidden
You do not have permission to perform this action. If this should not have occurred and the issue persists, please report it to support@osf.io.
```

**Comment by lbanner** [comment]
@mr-c Did you get that error trying to utilize one of the add-on credentials? Or trying to access them in general? Thanks -

**Comment by mr-c** [comment]
Attempting to view the content at https://osf.io/m2hig/wiki/home/ via a web browser produces the error quoted. I just confirmed using Google Chrome in incognito mode.

**Comment by lbanner** [comment]
@mr-c ... those credentials are for internal OSF devs only, which is why you get the 'forbidden'  -- Can you let me know where you found this text? In our docs? -- we'll want to update that.

**Comment by mr-c** [comment]
Ah.

As mentioned in the subject of this issue, in the last paragraph of https://github.com/CenterForOpenScience/osf.io/blob/develop/README.md

**Comment by lbanner** [comment]
Thanks @mr-c .... I apologize for the confusion.

**Comment by abought** [comment]
Should be fixed as of c9521fd .

**Comment by mr-c** [comment]
Fixed, thanks!
